hr_employee_spouse/models/res_partner.py

```diff
@@ -0,0 +1,59 @@
+# -*- coding: utf-8 -*-
+# © 2017 Therp BV <http://therp.nl>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openerp import _, api, models
+from openerp.exceptions import Warning as UserError
+
+
+class ResPartner(models.Model):
+    _inherit = 'res.partner'
+
+    def spouse_check(self, vals):
```

Comment:
i still think we don't need this

Reply:
we have agreed to see if client likes/wants it.